src/components/ebay-button/index.marko

```diff
@@ -40,6 +41,8 @@ $ {
     var fixedHeightClass =
         input.fixedHeight &&
         (sizeClass ? sizeClass + "-fixed-height" : baseClass + "--fixed-height");
+    var variant = input.variant || 'standard';
+    var variantClass = variant !== 'standard' ? `${baseClass}--${input.variant}` : "";
```

Comment:
Sorry one more nit. 
We can just do: 
```
`var variantClass = variant !== 'standard' && `${baseclass}--etc`;
```
If variantClass is undefined then it does not get added as a class in marko.